support-frontend/assets/components/headingBlock/headingBlock.scss

```diff
@@ -25,20 +25,17 @@
   }
 }
 
-.component-heading-block__overheading {
+.component-heading-block__overheading,
+.component-heading-block__overheading--thankyou {
```

Comment:
I think this might only need the modifier class in one place, if all you need is the background colour you wouldn't need `.component-heading-block__overheading--thankyou` here, all you would need to do is separate component-heading-block__overheading and overheadingClass```{`component-heading-block__overheading ${overheadingClass}`}``` and change overheadingClass to `.component-heading-block__overheading--thankyou`